content/javascript/concepts/arrays/terms/findLast/findLast.md

```diff
@@ -0,0 +1,81 @@
+---
+Title: '.findLast())'
+Description: "Finds the first value within the array, starting the search from the end of the array to the beginning of the array"
```

Comment:
```suggestion
Description: 'Returns the first value found in the array by starting the search from the end of the array to the beginning of the array.'
```
The initial phrasing made it sound it was finding the first element in the array.